python/tvm/contrib/graph_executor.py

```diff
@@ -212,7 +212,17 @@ def set_input_zero_copy(self, key, value, **params):
            Additional arguments
         """
         self._set_input_zero_copy(key, value)
-        self.set_input(None, None, **params)
+        if params:
```

Comment:
`key, value` vs `params` are basically two ways to specify the input. if `params` is supported, `key, value` should be by default None (similar to `set_input`)